From ecashin@coraid.com Wed Sep 20 11:53:04 2006
Message-ID: <be6d7c3099ce9b8b16f68de51699e78e@coraid.com>
Date: Wed, 20 Sep 2006 14:36:48 -0400
To: linux-kernel@vger.kernel.org
Cc: ecashin@coraid.com, Greg K-H <greg@kroah.com>
Subject: aoe: remove unused NARGS enum
From: "Ed L. Cashin" <ecashin@coraid.com>

The NARGS enum is left over from older code versions.

Signed-off-by: "Ed L. Cashin" <ecashin@coraid.com>
Acked-by: Alan Cox <alan@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/block/aoe/aoechr.c |    1 -
 1 file changed, 1 deletion(-)

--- gregkh-2.6.orig/drivers/block/aoe/aoechr.c
+++ gregkh-2.6/drivers/block/aoe/aoechr.c
@@ -15,7 +15,6 @@ enum {
 	MINOR_INTERFACES,
 	MINOR_REVALIDATE,
 	MSGSZ = 2048,
-	NARGS = 10,
 	NMSG = 100,		/* message backlog to retain */
 };
 
